envelope.md

```diff
@@ -52,14 +52,52 @@ envelopes with individual signatures.
 }
 ```
 
+### Signature Extensions
+
+In addition to `keyid` and `sig`, a signature object may include an `extension`
+field to store ecosystem specific information pertaining to the signature.
+Extensions do not modify the signing workflow established in the
+[DSSE protocol](protocol.md). However, signers and verifiers may exchange
+information about the extensions used out-of-band.
+
+```json
+{
+  "payload": "<Base64(SERIALIZED_BODY)>",
+  "payloadType": "<PAYLOAD_TYPE>",
+  "signatures": [{
+    "keyid": "<KEYID>",
+    "sig": "<Base64(SIGNATURE)>",
+    "extension": {
+      "kind": "<EXTENSION_KIND>",
+      "ext": {...}
+    }
+  }]
+}
+```
+
+`extension.kind` is a string and the `EXTENSION_KIND` value must unambiguously
+identify the type of the extension. This in turn identifies the fields in the
```

Comment:
What is the process for obtaining such a string?  

Do we track the set of assigned ones somewhere?

Where are the well-known ones defined?  Can we point to this location in the repo?

Reply:
I've added some text to extensions.md. I've also linked to that page here.